ci-operator/config/openshift/kubernetes/openshift-kubernetes-master.yaml

```diff
@@ -568,6 +568,14 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-external-aws
+- as: k8s-e2e-aws-hpa
+  optional: true
+  steps:
+    cluster_profile: aws-2
+    env:
+      KUBE_E2E_TEST_ARGS: -focus=\[Feature:HPA\]
+      NODES: "3"
+    workflow: openshift-kubernetes-e2e-aws
```

Comment:
In the spirit of saving CI resources, would it be possible to create a periodic job instead?

I'm not sure if it makes sense to run these tests/job every time we create a PR against o/k. IMO a periodic job would make more sense.

CC @jacobsee

Reply:
Sure, totally makes sense to me. I'll change it to a periodic with weekly frequency.